com.ultimaker.cura.yml

```
@@ -0,0 +1,313 @@
+        commit: baec470ec51f334f6bc5b275ddb946083970aac5
+        tag: 4.12.1
+      - type: patch
+        path: patch/pynest2d-python-install.patch
+
+  - name: savitar
+    buildsystem: cmake
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DCMAKE_PREFIX_PATH=/app
+      - -DCMAKE_INSTALL_PREFIX=/app
+      - -DCMAKE_CXX_STANDARD=17
+      - -DBUILD_STATIC=ON
+    sources:
+      - type: git
+        url: https://github.com/ultimaker/libSavitar.git
+        commit: 68a8868e0b30883b461c0dde77f6e85ce0a4d2ef
+        tag: 4.13.0
+      - type: patch
+        path: patch/libsavitar-python-install.patch
+
+  - name: libCharon
+    buildsystem: cmake
+    config-opts:
+      - -DCMAKE_PREFIX_PATH=/app
+      - -DCMAKE_INSTALL_PREFIX=/app
+      - -DPYTHON_EXECUTABLE=/app/bin/python3
+    sources:
+      - type: git
+        url: https://github.com/Ultimaker/libCharon
+        commit: 8e255aaad10b09c9cfc6e795f33968fd642e26c9
+        tag: 4.13.0
+
+  - name: cura-engine
+    buildsystem: cmake
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DCMAKE_PREFIX_PATH=/app
+      - -DCMAKE_INSTALL_PREFIX=/app
+      - -DCMAKE_VERBOSE_MAKEFILE:BOOL=ON
+      - -DCMAKE_CXX_STANDARD=17
+      - -DCURA_ENGINE_VERSION=4.13.0
+      - -DENABLE_MORE_COMPILER_OPTIMIZATION_FLAGS=ON
+    sources:
+      - type: git
+        url: https://github.com/ultimaker/CuraEngine
+        commit: 0e79559492a9e3e631c28624b73a888b3297f174
+        tag: 4.13.0
+
+
+  - name: uranium
+    buildsystem: cmake
+    config-opts:
+      - -DCMAKE_PREFIX_PATH=/app
+      - -DCMAKE_INSTALL_PREFIX=/app
+    sources:
+      - type: git
+        url: https://github.com/ultimaker/Uranium
+        commit: b1d9219a7272140a2021e7339901897c7df927d5
+        tag: 4.13.0
+
+  - name: fdm_materials
+    buildsystem: cmake
+    config-opts:
+      - -DCMAKE_PREFIX_PATH=/app
+      - -DCMAKE_INSTALL_PREFIX=/app
+    sources:
+      - type: git
+        url: https://github.com/ultimaker/fdm_materials
+        commit: 304287c3a69d79a14d33c1fe897bf653a2fc4f15
+        tag: 4.13.0
+
+  - name: cura
+    buildsystem: cmake
+    config-opts:
+      - -DCMAKE_PREFIX_PATH=/app
+      - -DCMAKE_INSTALL_PREFIX=/app
+      - -DCURA_VERSION=4.13.0
+      - -DCURA_BUILDTYPE=Release
+      - -DCURA_CLOUD_API_ROOT=https://api.ultimaker.com
+      - -DCURA_CLOUD_API_VERSION=1
+      - -DCURA_CLOUD_ACCOUNT_API_ROOT=https://account.ultimaker.com
+      - -DCURA_DIGITAL_FACTORY_URL=https://digitalfactory.ultimaker.com
+      - -DCURA_MARKETPLACE_ROOT=https://marketplace.ultimaker.com
+      - -DCURA_NO_INSTALL_PLUGINS='SliceInfoPlugin'
+      - -DURANIUM_SCRIPTS_DIR=
+    post-install:
+      - mv /app/share/mime/packages/cura.xml /app/share/mime/packages/com.ultimaker.cura.xml
+    sources:
+      - type: git
+        url: https://github.com/ultimaker/Cura
+        commit: c328f3619dae3ad37fb1decccf1013afa18b410c
+        tag: 4.13.0
+
+
+  - name: cura-binary-data
+    buildsystem: cmake
```

```suggestion
    buildsystem: cmake-ninja
```

unless it fails, it's faster